src/app/modules/main/profile_section/wallet/accounts/model.nim

```diff
@@ -108,6 +110,8 @@ QtObject:
       result = newQVariant(item.getPosition())
     of ModelRole.KeycardAccount:
       result = newQVariant(item.keycardAccount())
+    of ModelRole.PreferredSharingChainIds:
+      result = newQVariant(item.preferredSharingChainIds())
```

Comment:
Here you could return the short names

Reply:
I need to check if I have access to network module in here, otherwise, doing what you propose can "break" the current structure. I'll look at it!

Reply:
I've discussed that with @Khushboo-dev-cpp and we don't see it makes sense to add network specific data into the account model. We could do a join between account and network models if it was necessary but in this case, as we only need 1 specifiy role, I think adding the expression role is enough.

Reply:
> we don't see it makes sense to add network specific data into the account model

but you're doing it here :)

Reply:
hey preferredSharingChainIds is account specific data not linked with the networks directly. its the way the preferred networks are stored on the db on the go side